### DISCUSSION TITLE
FreeStyle Libre 3 sensor error on synamic drops / rises

### DISCUSSION THREAD

**Discussion by zibra** [j-kaltes/Juggluco#98] [General]
FreeStyle Libre 3 / Juggluco is giving sensor error on rapid BG drops or when sugar level is growing fast. These are most critical monents. Is it because of sensor not sending data? Is is possible to improve this somehow, or there is simply no data from the sensor in such moments?

**Comment by j-kaltes** [comment]
A sensor error in Libre 3 sensors is generated in the sensor itself, not by the software running on the phone or watch. This is different from Libre 0 and 2 sensors. There the software on the phone judges the glucose readings to be unreliable and doesn't show a glucose value because of that. Glimp, for example, does show glucose values when the Abbott’s Librelink app shows a sensor error. Glimp shows a value, but the values it shows diverges widely from the values shown by Abbott and they are much worse. They probably don’t use the factory calibration of Abbott. See: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5444502
